network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/text_only_teaser_block.html

```diff
@@ -10,20 +10,24 @@
                     <div class="tw-pt-8 tw-relative tw-w-full tw-flex tw-flex-col tw-border-t tw-border-black/10">
                         {% if card.link_page %}
                             <a class="hover:tw-text-inherit" href="{% pageurl card.link_page %}">
-                            {% elif card.link_url %}
-                                <a class="hover:tw-text-inherit"
-                                   target="_blank"
-                                   rel
-                                   noopener
-                                   href="{{ card.link_url }}">
-                                {% endif %}
                                 <div class="tw-h5-heading tw-mb-4">{{ card.heading }}</div>
-                                {% if card.link_page or card.link_url %}</a>{% endif %}
-                            {% if card.meta_data %}<div class="tw-body-small tw-pb-4">{{ card.meta_data }}</div>{% endif %}
-                            <p class="tw-text-sm tw-text-gray-60 tw-my-0">{{ card.body }}</p>
-                        </div>
-                    {% endwith %}
-                {% endfor %}
-            </div>
-        </section>
-    {% endblock %}
+                            </a>
+                        {% elif card.link_url %}
+                            <a class="hover:tw-text-inherit"
+                               target="_blank"
+                               rel
+                               noopener
+                               href="{{ card.link_url }}">
+                                <div class="tw-h5-heading tw-mb-4">{{ card.heading }}</div>
+                            </a>
+                        {% else %}
+                            <div class="tw-h5-heading tw-mb-4">{{ card.heading }}</div>
+                        {% endif %}
```

Comment:
I had to reformat this manually. There is some code duplication now but I think it's mode readable